src/fs/rock/RockRebuild.h

```diff
@@ -86,9 +86,18 @@ class Rebuild: public AsyncJob
     sfileno validationPos; ///< index of the loaded db slot being validated now
     MemBuf buf; ///< space to load current db slot (and entry metadata) into
 
+    // Balance our desire to maximize the number of entries processed at once
+    // (and, hence, minimize overheads and total rebuild time) with a
+    // requirement to also process Coordinator events, disk I/Os, etc.
+    static const int MaxSpentMsec = 50; // keep small: most RAM I/Os are under 1ms
+    static const int ForegroundNotificationMsec = 1000; ///< time interval to react to signals if opt_foreground_rebuild
+
```

Comment:
Since these constants are duplicated in UFS and Rock, let's move all of them into store_rebuild.h so that both modules can share them. Please see another change request regarding naming/comments, but the names will probably need a common Rebuild prefix. You may be able to use constexpr to avoid creating "global" statics?